x/execution/internal/keeper/keeper.go

```diff
@@ -291,6 +269,14 @@ func (k *Keeper) Update(ctx sdk.Context, msg types.MsgUpdate) (*executionpb.Exec
 		return nil, sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "no execution result supplied")
 	}
 
+	exec.Start = msg.Start
+	exec.Stop = msg.Stop
```

Comment:
I definitely suggest using elapsed time rather than saving the beginning and the end timestamp.

Reply:
see comment here https://github.com/mesg-foundation/engine/pull/1842#discussion_r432919324